searchcore/src/vespa/searchcore/proton/matching/matcher.cpp

```diff
@@ -311,7 +313,8 @@ Matcher::match(const SearchRequest &request, vespalib::ThreadBundle &threadBundl
         std::lock_guard<std::mutex> guard(_statsLock);
         _stats.add(my_stats);
         if (my_stats.softDoomed()) {
-            LOG(info, "Triggered softtimeout limit=%1.3f and duration=%1.3f", softLimit.sec(), duration.sec());
+            LOG(info, "Triggered softtimeout limit=%1.3f and duration=%1.3f, rankprofile=%s",
```

Comment:
Consider change wording to avoid confusion that softtimeout is actually happening.